_articles/til-persona-based-prompting.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'TIL: Persona-based Prompting'
+excerpt: "Stop asking AI for generic responses and start telling it *who* to be. Let's explore how persona-based prompting gives your AI interactions personality and character."
+date: '2025-04-22'
+slug: 'til-persona-based-prompting'
+authors:
+  - name: Sean Callan
+    github: https://github.com/doomspork
+tags:
+  - 'LLM'
+  - 'Prompt Engineering'
+  - 'TIL'
+---
+
+# TIL: Persona-based Prompting
+
+Have you ever wanted to spice up your LLM responses or wondered how your favorite TV character might respond to a particular question? Well you're in luck! Today, we'll explore a technique to achieve just that: **Persona-based Prompting**.
+
+## What is Persona-based Prompting?
+
+At its core, persona-based prompting involves instructing an LLM to adopt a specific character or personality when generating its response. Instead of just telling it *what* to do, you tell it *who* to be while doing it.
+
+## Isn't that just Role-based Prompting?
+
+Yes and no. It's similar, but often more specific. Let's do a quick refresher on Role-based Prompting then jump into personas.
+
+### Role-based Prompting Refresher 
```

Comment:
I wonder if this and the section below should be `##` 🤔

Reply:
I think it looks fine, at least here on github